test/blackbox-tests/test-cases/tests-stanza/github6132.t

```diff
@@ -0,0 +1,19 @@
+  $ cat >dune-project <<EOF
```

Comment:
Could you add a description?

Reply:
I don't see a description for this test yet.

Reply:
The description of the expected result is line 19, just before the actual running of the test. Did you have something else in mind ?